### PR TITLE
Workbook sentinel coverage

### PR DESCRIPTION
**Pull request by anders-alex**
Required items, please complete
   
   Change(s):
   - Added new workbook: Azure Log Coverage

   Reason for Change(s):
   - There was not an existing workbook for this.

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @anders-alex can you please provide sample data for the workbook, The table which we already have does not contain any column named as queryTa 
image

**Comment by anders-alex** [comment]
@v-prasadboke - The workspace needs to be enabled for Traffic Analytics for this section of the workbook to function (I noted that in the information text box). The table gets created once Traffic Analytics is enabled is: AzureNetworkAnalytics_CL.

**Comment by v-prasadboke** [comment]
Ok @anders-alex going through this and please resolve branch conflicts

**Comment by v-prasadboke** [comment]
Hello @anders-alex working in this

File: Workbooks/AzureLogCoverage.json [hunk 1]
```diff
@@ -0,0 +1,1761 @@
...
+      },
+      "conditionalVisibility": {
+        "parameterName": "parameterTab",
+        "comparison": "isEqualTo",
+        "value": "law"
+      },
+      "name": "groupLAW"
+    }
+  ],
+  "fromTemplateId": "sentinel-UserWorkbook",
```
Comment (v-prasadboke) [inline Workbooks/AzureLogCoverage.json:1759]: Hello @anders-alex please change the templateid from "sentinel-UserWorkbook" to any other id.

 Please be consistent with the format ' sentinel-"workbookName" '

**Comment by v-prasadboke** [comment]
> @v-prasadboke - The workspace needs to be enabled for Traffic Analytics for this section of the workbook to function (I noted that in the information text box). The table gets created once Traffic Analytics is enabled is: AzureNetworkAnalytics_CL.

Can you please share some working of workbook images for the same

**Comment by v-prasadboke** [comment]
Hello @anders-alex any updates

**Comment by anders-alex** [comment]
@v-prasadboke - here you go: 
image

File: Workbooks/AzureLogCoverage.json [hunk 2]
```diff
@@ -0,0 +1,1761 @@
...
+              "additionalResourceOptions": [
+                "value::1"
+              ],
+              "showDefault": false
+            },
+            "defaultValue": "value::1",
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources",
+            "value": [
+              "/subscriptions/8a27ecf5-1c40-4eee-a29d-0a7f9a7de45f/resourcegroups/rg-monitor-prd-scu/providers/microsoft.operationalinsights/workspaces/log-sentinel-prd-scu-1"
```
Comment (v-prasadboke) [inline Workbooks/AzureLogCoverage.json:102]: Hello @anders-alex please keep the value null as "value" : [ ]

**Comment by anders-alex** [comment]
@v-prasadboke - anything else needed from me to get this published?

**Comment by v-vdixit** [comment]
Hi @anders-alex we need to fix all the validation failures before getting this published, can you please take the latest from master and run validation checks again, thanks!

File: Workbooks/AzureLogCoverage.json (hunk 1 above)
Comment (v-prasadboke) [reply Workbooks/AzureLogCoverage.json]: > Hello @anders-alex please change the templateid from "sentinel-UserWorkbook" to any other id. There is already one workbook with the same templateID
> 
> Please be consistent with the format ' sentinel-"workbookName" '

still not resolved

**Review by v-vdixit — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**